build.gradle

```diff
@@ -243,6 +243,9 @@ project('reactor-extra') {
   dependencies {
 	optional "org.eclipse.swt:org.eclipse.swt.${getPlatform()}:${swtVersionPlatform}"
 	optional("org.jetbrains.kotlin:kotlin-stdlib:${kotlinVersion}")
+	optional 'org.openjfx:javafx-base:12-ea+7'
+    	optional 'org.openjfx:javafx-graphics:12-ea+7'
+	optional 'org.openjfx:javafx-controls:12-ea+7'
```

Comment:
do we really need controls? is there a way to avoid using Animation (and thus also remove the `graphics` dependency)? The less dependency the better... For instance the `SwingScheduler` uses a plain `Timer`.

Reply:
I now made use of a java.util.Timer, but the SwingScheduler class actually uses a swing specific class as well: javax.swing.Timer.

Reply:
(actually it uses a _Swing_ `Timer`, but a shared `java.util.Timer` could do)